README.md

```diff
@@ -6,11 +6,42 @@ to allow codebases already using AWS credentials to use IAM roles with minimal t
 refactoring. In the same vein, the Role Injector can also be used to help users using the
 command line tools to assume a role.
 
+## Assumptions
+ - AWS CLI configured correctly, storing 'aws_access_key_id' and
+   'aws_secret_access_key' in either environment variables -OR- in
+   ~/.aws/credentials
+ - One of the following Scenarios apply:
+
+### Scenario One: Federated AWS Accounts
+ - At least two AWS Accounts:
+   - AWS Account 1 must have a policy that includes sts:AssumeRole to AWS Account 2
+   - AWS Account 2 must have a Trust Relationship on a role that references AWS Account 1
+ - AWS Account 1 may now assume the the role in AWS Account 2 that has the Trust Relationship
+
+### Scenario Two: Single AWS Account:
+ - An IAM User Account with a policy that include sts:Assume on a IAM
+   Role.
+ - The IAM Role has a policy that allows the IAM User to assume it
```

Comment:
s/policy that allows the IAM User/trust relationship that allows entities in the account